common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -668,9 +839,9 @@ public void onData(String streamId, ByteBuffer buf) throws IOException {
             return;
           }
           abortIfNecessary();
-          logger.trace("{} shuffleId {} reduceId {} onData writable",
-            partitionInfo.appId, partitionInfo.shuffleId,
-            partitionInfo.reduceId);
+          logger.trace("{} shuffleId {} shuffleMergeId {} reduceId {} onData writable",
+            partitionInfo.appId, partitionInfo.shuffleId, partitionInfo.shuffleMergeId,
+              partitionInfo.reduceId);
```

Comment:
Extract this common pattern into the `toString` method of `AppShufflePartitionInfo`?

Reply:
Can we do this separately in a follow-up PR?

Reply:
Sure. But please file a JIRA ticket for it:)

Reply:
https://github.com/apache/spark/pull/33561 - created a WIP PR based off of this PR. Will fix that once it gets merged.